R/plot_compare_stage.R

```diff
@@ -1,10 +1,9 @@
-#'@title Plot simulated and observed stage data
+#'Plot simulated and observed stage data
```

Comment:
Dropping the title tag makes it so the text is included in the `title` as well as the `description` to satisfy `devtools::check`. Alternatively, I could have added a `@description` tag but I think this way is cleaner.